### DISCUSSION TITLE
Could Markor recover to the last read markdown when startup?

### DISCUSSION THREAD

**Discussion by dusk1009** [gsantner/markor#1524] [Q&A] [answered]
hi,  thanks for the useful tool first.
I use markor to read markdown books in view mode.  But it's not too convenient to remember the last read.  
Is there a way to add bookmark when read or could Markor recover to the last read markdown when startup?

**Comment by gsantner** [comment, marked as answer]
Markor settings has a option to configure the "app start folder" of the file browser. File browser has a special "recent" folder, and you can select it here.

Then always when you start Markor, the app starts with the list of recent opened files.

PRO TIPP / secret feature: You can also long-press the "red +" button at filebrowser, and it jumps to recents. :wink:

**Reply by dusk1009** [reply]
thanks for your quick reply. 😄

But the recent folder is sorted by modify date not read date.  And it would be better if markor can jump to the last read screen when click the file.

**Reply by gsantner** [reply]
> But the recent folder is sorted by modify date not read date. 

This is something I'm aware off - and somebody who works on it / fixes it.

 > And it would be better if markor can jump to the last read screen when click the file.

Markor remembers last state. If you quit with the view mode on (and not change to edit), view mode will be opened again next time you open the file.

**Reply by dusk1009** [reply]
> And it would be better if markor can jump to the last read screen when click the file.

what I mean is when I start to read a file, markor will scroll down to the last place I read, not begin of the file.  It's like a bookmark to remember read state in every file.

**Reply by gsantner** [reply]
<https://github.com/gsantner/markor/issues/1025>

In case you know how to, feel free to do it

**Reply by dusk1009** [reply]
So it is. Thank you. 
I'm not good at mobile development.😂
